platform/nodejs/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "description": "To create a working instance of uBlock Origin's static network filtering engine",
   "type": "module",
-  "main": "main.js",
+  "main": "index.js",
```

Comment:
The rationale for this change is that the benchmarking code should have to know as little as necessary about the internals of this package. By using the default name, we make this easier. If this wasn't `"type": "module"` there would be no need for a `main`  field either.